case_study/memory_leaks/leaking_counter_1/README.md

```diff
@@ -4,36 +4,21 @@ This is memory leaking application to test memory debugging tools.
 
 ## Leak debugging lab
 
-Users report that the app consumes more and more memory with every button click.
-Your task is to fix the leak using the tab Memory > Diff, assuming the application is too large to find the issue by
-simply reviewing the code of the button handler.
-
-Setup the lab:
-
-1. Clone DevTools: `git clone git@github.com:flutter/devtools.git`
-
-2. Navigate to the DevTools directory (`cd devtools`) in two console tabs
+1. Setup the lab:
 
-3. In the first console tab:
+    a. Start the latest version of DevTools by following [BETA_TESTING.md](https://github.com/polina-c/devtools/blob/master/BETA_TESTING.md)
 
-    a. Start the app:
+    b. in another console tabs navigate to the DevTools directory (`cd devtools`) and start the app:
 
-    ```
-    cd case_study/memory_leaks/leaking_counter_1
-    flutter run -d macos --profile
-    ```
+        ```
+        cd case_study/memory_leaks/leaking_counter_1
+        flutter run -d macos --profile
+        ```
 
-    b. Copy the Observatory URL displayed in the console
+    c. Copy the Observatory URL displayed in the console to the connection box in DevTools
```

Comment:
We shouldn't reference Observatory anymore since we're going to be removing it. Change this to "Copy the Dart VM Service URL...".

Reply:
this is the console output:

```
💪 Running with sound null safety 💪

An Observatory debugger and profiler on macOS is available at: http://127.0.0.1:61713/PM2kWICS5cw=/
The Flutter DevTools debugger and profiler on macOS is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:61713/PM2kWICS5cw=/
```

If we say `the Dart VM Service URL`, non experienced users will be confused and lost. So, for now I would use as it is worded in flutter tools.
As soon as the console output changes, we will update the documentation.

Reply:
Ugh, I thought Flutter Tools was already updated not to reference Observatory... I'll need to fix that :-(